ecs-proxies-deploy.yml

```diff
@@ -15,6 +15,8 @@ docker_service:
         value: ou=services,o=nhs
       - name: MHS_SDS_REDIS_CACHE_HOST
         value: cache-service
+      - name: MHS_SDS_REDIS_DISABLE_TLS
```

Comment:
Where did this come from?  Please can you remove it.

Reply:
Hmm, I think it is some merging effect, I saw it in conflict and thought it is something necessary

Reply:
removed